packages/compat/src/Table.ts

```diff
@@ -189,6 +189,7 @@ enum TableFocusTargetElement {
 		*/
 		row: { type: HTMLElement },
 	},
+	bubbles: true,
```

Comment:
I do not get why the row-click event that is fired from the table supposed to bubble?

Reply:
because the old `fireEvent` method was creating events with `bubbles: true` unless a parameter was specified, and this is the compatible replacement.
```ts
	fireEvent<T>(name: string, data?: T, cancelable = false, bubbles = true): boolean {
```

other than that, the `row-click` event is actually coming from the `TableRow.ts` via bubbling, maybe it was documented here, as event delegation makes more sense. It could be changed as you described, but this change was targeted at better describing the existing behavoiur, not so much about refactoring.